src/__test__/actions/rating.action.test.js

```diff
@@ -0,0 +1,116 @@
+import configureStore from 'redux-mock-store';
+import axios from 'axios';
+import MockAdapter from 'axios-mock-adapter';
+import thunkMiddleware from 'redux-thunk';
+import promiseMiddleware from 'redux-promise-middleware';
+import {
+  averageRating,
+  rateArticle,
+  failedRating,
+  rateArticleAction,
+  getRatings,
+} from '../../actions/rating.action';
+import ARTICLE from '../../mock/article';
+
+const middlewares = [
+  thunkMiddleware,
+  promiseMiddleware(),
+];
+const mockStore = configureStore(middlewares);
+const store = mockStore({});
+
+describe('rate article action', () => {
+  afterEach(() => {
+    store.clearActions();
+  });
+
+  it('rate article action received valid data', () => {
+    store.dispatch(rateArticle({
+      avg_rating: 2,
+      rating: 3,
+    }));
+    const expectedOutput = [{
+      payload: {
+        averageRating: 2,
+        rating: 3,
+      },
+      type: 'RATE_ARTICLE',
+    }];
+    expect(store.getActions()).toEqual(expectedOutput);
+  });
+
+  it('rate article action receives no data. should return 0', () => {
+    store.dispatch(rateArticle({
+      avg_rating: null,
+      rating: null,
+    }));
+    const expectedOutput = [{
+      payload: {
+        averageRating: 0,
+        rating: 0,
+      },
+      type: 'RATE_ARTICLE',
+    }];
+    expect(store.getActions()).toEqual(expectedOutput);
+  });
+});
+
+describe('failedRating action. It should update the state with the error ', () => {
```

Comment:
I don't think you should have so many `describe` blocks in one file. Just make sure you give the `it` a descriptive title and it should make sense. Otherwise, you just keep repeating this
```
afterEach(() => {
    store.clearActions();
  });
```